package.json

```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "@fortawesome/fontawesome-free": "5.15.2",
-    "@universityofwarwick/bootstrap-3-typeahead": "4.1.0",
+    "@universityofwarwick/bootstrap-3-typeahead": "4.1.1",
```

Comment:
As discussed, could add `^` to allow apps to updoot to any 4.x.x version without needing to do this again

Reply:
i shall close this pr as not needed for now